harper-core/src/linting/was_aloud.rs

```diff
@@ -0,0 +1,89 @@
+use super::{Lint, LintKind, PatternLinter};
+use crate::linting::Suggestion;
+use crate::patterns::{Pattern, SequencePattern, WordSet};
+use crate::Token;
+use crate::TokenStringExt;
+
+pub struct WasAloud {
+    pattern: Box<dyn Pattern>,
+}
+
+impl Default for WasAloud {
+    fn default() -> Self {
+        let pattern = SequencePattern::default()
+            .then_word_set(WordSet::all(&["was", "were"]))
```

Comment:
Don't forget the passive can also be used in the present and perfect:
- You may **be aloud** to enter if you ask permission first.
- If I had **been aloud** to enter I would've jumped at the chance.

Also, adverbs can be interposed such as "not", "always", "usually", "often", etc.